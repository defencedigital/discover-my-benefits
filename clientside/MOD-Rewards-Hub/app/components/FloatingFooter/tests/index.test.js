import React from 'react';
import { fromJS } from 'immutable';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { FloatingFooter } from '../index';
import { service, question, commitmentType, option } from '../../../tests/commonMocks';

Enzyme.configure({ adapter: new Adapter() });

const mockedProps = fromJS({
  service,
  questions: [question],
  commitmentTypes: [commitmentType],
  options: [option],
});

describe('FloatingFooter component', () => {
  it('matches the snapshot', () => {
    const html = shallow(<FloatingFooter {...mockedProps.toJS()} />);
    expect(html).toMatchSnapshot();
  });
});
