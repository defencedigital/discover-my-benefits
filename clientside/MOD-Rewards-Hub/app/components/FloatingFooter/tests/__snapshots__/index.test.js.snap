// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatingFooter component matches the snapshot 1`] = `ShallowWrapper {}`;
